### PR TITLE
chore(code): use prettier for consistent indenting

### PR DESCRIPTION
**Pull request by Haroenv**
I noticed that some things changed (tabs/spaces) when opening it in my editor, so I set up prettier for all typescript and javascript files. I noticed single quotes were preferred, so I set that setting in prettier.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +32,10 @@
     "autoprefixer": "^7.1.1",
     "google-closure-compiler": "^20170521.0.0",
     "http-server": "^0.10.0",
+    "husky": "^0.13.4",
```
Comment (samccone) [inline package.json]: oh neat!

**Comment by Haroenv** [comment]
Sure, that would be using `prettier-eslint`. I should take a look at how react did it.

**Comment by Haroenv** [comment]
@samccone I added a test, but I’m not 100% sure if it works correctly, could you enable travis on pull requests? I think it’s disabled

**Comment by samccone** [comment]
Hey @Haroenv I plan on landing this and all the big TS work this weekend, to minimize thrashing with rebasing :) 

thanks so much for your prs thus far.

**Comment by Haroenv** [comment]
Sure. Lmk when to rebase

**Comment by samccone** [comment]
Landed as
a7a8c6b2cea081c403bbff75f85d948e050547ca

Thanks again for your help here... super awesome

**Pull request closed without merging**